nipoppy/config/pipeline.py

```diff
@@ -4,7 +4,7 @@
 
 from abc import ABC
 from pathlib import Path
-from typing import Any, Optional, Type, Union
+from typing import Any, Optional, Union
```

Comment:
I think if you use `from __future__ import annotations` then you would be able to use the `|` operator instead of `Union` 🍒